### DISCUSSION TITLE
심화프로젝트 2주차 팀회고

### DISCUSSION THREAD

**Discussion by sulyoungoh** [codeit-bootcamp-frontend/3-Weekly-Mission#346] [회고]
# 왜 회고를 해야하나요?
- **원활한 팀 프로젝트 진행을 위해, 회고가 좋은 도구가 되어줄거예요.** 
매주 팀 단위로 회고를 하며 어떤 점이 좋았고, 어떤 점이 아쉬웠고, 아쉬웠던 부분은 어떻게 개선해볼지 함께 이야기해보면 
그 다음주에는 더 나은 프로젝트를 진행할 수 있을거예요. 회고를 통해 함께 성장해보아요 🔥 
- **회고 시간만큼은 솔직한 생각, 마음을 터놓아보기! 😎** 
솔직한 회고를 통해 다음주는 더 즐겁게 프로젝트를 진행할 수 있을거예요.  

# 회고 방법
![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/144666341/b1f1cc83-7d24-4156-910d-3fc9b6bb7555)
회고는 **“KPT 회고”** 방식으로 진행해볼거예요. 아래 방법대로 회고해주시면 됩니다. 
- 팀원들과 모여 그 주의 팀 프로젝트 활동을 돌아보며, '좋았던 점(Keep) / 아쉬웠던 점(Problem) / 시도해볼 것(Try)'으로 항목을 나누어 회고해주세요.
    - Keep: 현재 우리팀이 잘 하고 있어서 계속 유지하면 좋을 것 같은 부분
    - Problem: 현재 우리팀이 가지고 있는 문제
    - Try: Problem에 적은 것을 해결하기 위해 우리 팀이 다음에 시도해볼만한 것 (혹은 Keep 중에서도 더 개선하거나 시도해보고 싶은 것)
- [KPT 회고 피그마 템플릿](https://www.figma.com/file/0NHIhlM9vhYZDrtZa7nJn8/%ED%9A%8C%EA%B3%A0-%ED%85%9C%ED%94%8C%EB%A6%BF?type=whiteboard&node-id=0%3A1&t=p6vXDVEjQntHKaNP-1)을 활용해서, 팀 회고를 진행하셔도 됩니다. 단 템플릿 사용시, 팀별로 복제해서 사용해주세요. 꼭 템플릿을 사용하지 않아도 되니, 팀별로 편한 방식으로 회고를 진행해주세요. 
- **이번주 일요일까지, 팀 회고한 내용을 정리하여 댓글로 올려주세요.** 
(댓글에 예시 회고를 올려드릴테니, 참고해주세요. 피그마 템플릿으로 진행하셨다면 캡처본을 올려주셔도 돼요.)

# 참고 글
- [KPT 회고란 무엇인가?](https://brunch.co.kr/@jinha0802/35)
- [팀문화의 탄생](https://techblog.woowahan.com/2677/)

**Comment by ohddang** [comment]
# 3팀

## K (Keep)
- 현업에서 실제 사용하는 기술 적용 keep going
- 도움될만한 공부자료들을 공유해주셔서 좋았습니다
- 데일리로 일정 및 한 일 체크
- 꾸준히 위워크에 나와 자주 소통하는 모습 좋습니다

## P (Problem)
- PR 리뷰
- 처음 접하는 부분이 많아서 많은 어려움이 있음
- 프론트 이외의 부분에서 도움 받기가 어려움

## T (Try)
- PR 리뷰 하나씩 달아보기
- 본인이 어려움을 겪고 있는 부분이나 해결한 내용 공유

**Comment by yunsunglee2** [comment]
# 7팀

## K (Keep)
- 협업 팀원들이 원활한 교류로 기획단계에서 큰 무리 없이 진행이 가능했다.
- 부가적인 기능, 주 기능을 나누어 먼저 구현해야하는 최소 사항을 고려했더니 진행에 무리가 없었다. 
- 미리 배포를 해두어 PR을 올릴때 빌드시 오류까지 고려하고 있다. 
- 정규 멘토링 이외의 자투리 멘토링으로 원활한 기획이 가능했다. 

## P (Problem)
- 컨벤션 지키려고 노력하자 
- 팀원과 작업하면서 겹치는 부분에 대해 관심갖고 소통하기 
- 라이브러리를 어느정도 숙지하고 제대로 써보기 

## T (Try)
- 너무 자주는 아니더라고 디자이너, 백엔드와의 정기적인 회의 일정을 잡아보는건 어떨지? 
- 팀원들의 PR에 관심갖고 코드리뷰 열심히 하기

**Comment by jinahp** [comment]
# 6팀
## K (Keep)
- 맡은 파트에 대한 책임감이 강해서 본인이 맡은 공통 컴포넌트를 팀원들이 잘 사용할 수 있도록 코드를 열심히 짜고 설명한다.  
- 우선 자기힘으로 해보고 진짜 안되겠을때만 ai에 물어본다.  
- 앞으로 최대한 코딩에 시간을 투자한다.  
- 팀원들과 적극적이고 능동적인 소통을 한다.  
- 다들 적극적으로 참여하는 부분이 좋았습니다!  
- 개인 플레이가 아니라, 자신 파트가 아니어도 같이 고민하고 풀어나가는 점이 좋았습니다!  
- 아이디어도 많이 나오고 모르는 부분, 알게 된 부분을 서로 공유하는 점이 너무너무 좋았습니다!

## P (Problem)
- 최대한 기초 설정을 탄탄하게 해놓고 시작했다고 생각하지만 이슈가 발생했다.  
- 취침시간,기상시간이 삐뚤어지기 시작했다.  
- 생각보다 일정이 타이트하여 뒤늦게 구현을 시작했다.  
- 과거에 만들었던 코드가 기억이 안난다.  
- 코드컨벤션을 정해놨는데 잘 지켜지지 않는..(?) 뭔가 모호한(?) 느낌이 듭니다!  
- 개인일정 때문에 기능 구현이 늦어졌습니다 ㅠㅠ.. 죄송합니다..ㅠㅠ  

## T (Try)
- 지금처럼 이슈가 발생할 때마다 어떤 이유에서 그런 상황이 생겼는지 열심히 회의하고, 이해 안 되는 점들을 서로 공유하면 좋겠다.
- 열두시 되면 폰 충전기에 꽂고 침대에 누워서 눈감고있기 (잠이 오든 안오든)
- 팀원들의 배려 잊지 말고, 폐 끼치지 않도록 열코하기!!
- 프로젝트 초기에 정해놨던 부분들을 주기적으로 상기해서 진행하면 좋겠습니다!
- 개인적으로 가끔씩이라도 게더타운이라던지 디코라던지 모각코를 해보는 것도 좋을 것 같습니다 ㅎㅎ

**Comment by Dev-Duke-Seo** [comment]
# 6팀
# K (Keep)
- 의견을 적극적으로 내고, 자신의 의견만 주장하지 않아서 좋은 것 같습니다.
- 긴 회의에도 웃음을 잃지 않는 모습이 보기 좋습니다.
- 다들 성실하게 참여하는 모습이 좋습니다.
- 팀원들간 다양한 의견을 주고받으며 같은 그림을 그려나가는게 좋습니다.
- 의견 충돌이 생겼을 때, 서로 의견을 맞춰나가는 모습이 좋았습니다.
- 다양한 의견 교환이 좋았습니다.

# P (Problem)
- 개발 일정이 늦어지는 것 같습니다.
- 회의가 길어집니다.
- 회의 내용을 제대로 마무리 짓지 않아 서로 알고 있는 부분이 다를 때가 많습니다.
- 프로젝트 규모를 고려했을 때 비교적 급하지 않은 문제도 걱정하며 논의가 길어질 때가 많습니다.
- 회의가 길어지다보니 집중력이 낮아집니다. 😭
- 했던 이야기를 반복해서 하는 일이 많습니다.
- 묻거나 요청한 부분에 대해서 대답이나 피드백이 없는 경우가 잦습니다.

# T (Try)
- 기술에 대한 공부를 좀 해야 할 것 같습니다.
- 회의를 짧게 가져갈 수 있도록 준비해오고 준비해온것만 순서대로 하면 좋겠다.
- 기획을 미리 꼼꼼히 하는 것도 중요하지만 개발도 빨리 시작해야할 것 같습니다.
- 진행상황을 자주 공유했으면 좋겠습니다.
- 어떤 주제를 가지고 회의를 할 지 미리 정하고, 회의 시작 전에 미리 생각해오면 좋겠습니다.
- 원활한 소통을 위해 모두의 노력이 필요한 것 같습니다.

**Comment by eundin** [comment]
## 2팀

### Keep
- 다들 의견 수용을 잘 해주신다.
- 서로 의사소통이 잘 된다.
- 누군가의 독단으로 의사결정을 하는 것이 아닌, 커뮤니케이션을 통해 정한다.
- 모두의 합의 하에 의사결정이 진행되면, 다들 수용을 잘 해주신다.

### Problem
- 조금 더 의견을 내도 좋을 것 같다.
- 아직 서로 조금 어색한 것 같다.
- 발언을 이끄는 사람이 없으면 조용해지는 등의 문제가 있어 더욱 적극적인 커뮤니케이션이 필요하다고 느낍니다.

### Try
- 각자 맡은 부분 열심히 하기
- 좀 더 의견을 내주셨으면 좋겠습니다
- 팀미팅 시간에 더욱 적극적으로 얘기해주셨으면 합니다
- 친해져요 우리...
